src/main/resources/openapi.json

```diff
@@ -753,6 +753,52 @@
                 }
             ]
         },
+        "/admin/topics": {
+            "post": {
+                "tags": [
+                    "Topics"
+                ],
+                "description": "Creates a topic with given name, partitions count and replication factor.",
+                "operationId": "createTopic",
+                "requestBody": {
+                    "description": "Creates a topic with given name, partitions count and replication factor.",
+                    "content": {
+                        "application/vnd.kafka.json.v2+json": {
+                            "schema": {
+                                "$ref": "#/components/schemas/NewTopic"
+                            }
+                        }
+                    },
+                    "required": true
+                },
+                "responses": {
+                    "201": {
+                        "description": "Created",
+                        "content": {
+                            "application/vnd.kafka.v2+json": {}
+                        }
+                    },
+                    "400": {
```

Comment:
I would remove this one because we don't have this kind of error described in any of the other operations (even if it can happen on the OpenAPI validation side).